CMakeLists.txt

```diff
@@ -860,14 +860,26 @@ endforeach()
 
 ########################################################################
 ## Populate the ZEEK_BUILD_INFO for use in src/version.c.in
-execute_process(COMMAND "${PROJECT_SOURCE_DIR}/ci/collect-repo-info.py" "${ZEEK_INCLUDE_PLUGINS}"
-                WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
-                OUTPUT_VARIABLE ZEEK_BUILD_INFO
-                RESULT_VARIABLE ZEEK_BUILD_INFO_RESULT
-                OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+if ( WIN32 )
+  # Windows installs Python to C:\Python311\python, but doesn't create a version
+  # or symlink to python3. Call python with the script directly since the shebang
+  # in the script won't work here.
+  execute_process(COMMAND "python" "${PROJECT_SOURCE_DIR}/ci/collect-repo-info.py" "${ZEEK_INCLUDE_PLUGINS}"
```

Comment:
Thanks.

I'm a bit confused about the indentation as above it's 4 and below it's 4 and here it's 2.